TESTS/mbed_hal/stack_size_unification/stack_size_unification.h

```diff
@@ -0,0 +1,49 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2017-2018 ARM Limited
```

Comment:
if we add new files, we should add SPDX identifier as well, can you please

Reply:
Fixed!